From 3747044a0d25f9b0ea3de86a91d1df505416f399 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Sat, 12 May 2018 03:24:55 +0200
Subject: [PATCH 1/1] README.uefi: update list of missing functionality

Update the list of missing functionality for boottime and runtime services.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 doc/README.uefi | 9 +++++++--
 1 file changed, 7 insertions(+), 2 deletions(-)

diff --git a/doc/README.uefi b/doc/README.uefi
index 2ad9217db85..6b9759cfede 100644
--- a/doc/README.uefi
+++ b/doc/README.uefi
@@ -299,13 +299,18 @@ This driver is only available if U-Boot is configured with
     CONFIG_BLK=y
     CONFIG_PARTITIONS=y
 
-## TODOs as of U-Boot 2018.03
+## TODOs as of U-Boot 2018.07
 
 * unimplemented or incompletely implemented boot services
   * Exit - call unload function, unload applications only
-  * ReinstallProtocolInterface
+  * ProtocolRegisterNotify
   * UnloadImage
 
+* unimplemented or incompletely implemented runtime services
+  * SetVariable() ignores attribute EFI_VARIABLE_APPEND_WRITE
+  * GetNextVariableName is not implemented
+  * QueryVariableInfo is not implemented
+
 * unimplemented events
   * EVT_RUNTIME
   * EVT_SIGNAL_VIRTUAL_ADDRESS_CHANGE
-- 
2.17.0

